runtime/js/01_web_util.js

```diff
@@ -27,17 +27,41 @@
   }
 
   const handlerSymbol = Symbol("eventHandlers");
-  function makeWrappedHandler(handler) {
+  function makeWrappedHandler(handler, isSpecialErrorEventHandler) {
```

Comment:
TIL about the special error handler handling. Yucky...

Instead of the `isSpecialErrorEventHandler` flag, could you just switch on `evt instanceof ErrorEvent && evt.type === "error" && target instanceof DedicatedWorkerGlobalScope`?

Reply:
I didn't want to do that because the `DedicatedWorkerGlobalScope` interface is not available in the main worker, but I could check for it as part of the condition.

Reply:
Right... never mind then. Let's keep as is.

Unrelated, but we should go and deduplicate `defineEventHandler` with the one in 02_event.js in extensions/web (available via `window.__bootstrap.event.defineEventHandler`) in a follow-up. I am about 70% through a rewrite of the 02_event.js file though, so I'll do this as part of that rewrite.